pkg/cli/importimage/importimage_test.go

```diff
@@ -597,11 +651,39 @@ func TestCreateImageImport(t *testing.T) {
 			o.InsecureFlagProvided = true
 		}
 
+		// set default for referencePolicy
+		if len(test.referencePolicy) == 0 {
+			for i := range test.expectedImages {
+				test.expectedImages[i].ReferencePolicy.Type = imagev1.SourceTagReferencePolicy
+			}
+			if test.expectedRepository != nil {
+				test.expectedRepository.ReferencePolicy.Type = imagev1.SourceTagReferencePolicy
+			}
+		}
+
+		// set default for importMode
+		if len(o.ImportMode) == 0 {
+			for i := range test.expectedImages {
+				test.expectedImages[i].ImportPolicy.ImportMode = imagev1.ImportModeLegacy
+			}
+			if test.expectedRepository != nil {
+				test.expectedRepository.ImportPolicy.ImportMode = imagev1.ImportModeLegacy
+			}
+		}
+
 		if err := o.parseImageReference(); err != nil {
 			t.Errorf("unexpected error: %v", err)
 			continue
 		}
 
+		if err := o.Validate(); err != nil {
```

Comment:
This makes sense. I've tweaked it a bit as retaining the behavior of `continue` is needed if there is a validation error so that the import object doesn't actually get created in the test. This makes it clearer what actually failed and emulates the actual behavior better. The general "unexpected error" case is already handled below.